### PR TITLE
Describe the view-component to load data-series

### LINKED ISSUE #55: View-component load_input_dataseries is not described yet
This is a view-component where the user can choose to upload arrays, .xls, .csv files as value for a parameter, this is linked to the view-component [input_parameter_field](https://open-plan.readthedocs.io/en/latest/tool_interface.html#input-parameter-field)

File exists

### PR DESCRIPTION
**Pull request by mahendrark**
Fix #55 

**Changes proposed in this pull request**:
- Your_changes

The following steps were realized, as well (if applies):
- [x] Apply black (`black . --exclude docs/`)
- [x] Update the CHANGELOG.md

The following optional steps were realized:
- [ ] Use in-line comments to explain your code
- [ ] Write docstrings to your code
- [x] For new functionalities: explain in readthedocs
- [ ] Write test(s) for your new patch of code

<sub>*For more information on how to contribute check the [CONTRIBUTING.md](https://github.com/rl
-institut/open_plan/blob/dev/CONTRIBUTING.md).*<sub>

### CODE REVIEW

**Comment by mahendrark** [comment]
@Bachibouzouk 

Its not displaying in the docs. I cant find the reason why. Can you please check

File: docs/view_components/load_input_dataseries.rst [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+
+Attributes
+^^^^^^^^^^
+
+**Label**
+    Name of the parameter which would be displayed close to the input field
+
+
+**Input field**
+
```
Comment (Bachibouzouk) [inline docs/view_components/load_input_dataseries.rst]:
```suggestion
```

File: docs/view_components/load_input_dataseries.rst [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+**Input Unit**
+    Unit of the input parameter
+
+    Actions index: 1
+
+    Requirements index: 2
+
+**Load File button**
+
```
Comment (Bachibouzouk) [inline docs/view_components/load_input_dataseries.rst]:
```suggestion
```

File: docs/view_components/load_input_dataseries.rst [hunk 3]
```diff
@@ -0,0 +1,85 @@
...
+    Actions index: 2
+
+    Requirements index:
+
+**Visualize Data button**
+
```
Comment (Bachibouzouk) [inline docs/view_components/load_input_dataseries.rst]:
```suggestion
```

**Review by Bachibouzouk — APPROVED** [review]
Thanks for this first draft @mahendrark

**Pull request merged**